pal/system-info/minigame/system-info.ts

```diff
@@ -61,6 +61,37 @@ if (WECHAT) {
     currentPlatform = Platform.QTT_MINI_GAME;
 }
 
+let isVersionGreaterOrEqualTo;
+if (BYTEDANCE) {
+    isVersionGreaterOrEqualTo = function isVersionGreaterOrEqualTo (versionA: string, versionB: string): boolean {
+        // Split the version number string into an array of integers
+        function parseVersion (version: string): number[] {
+            return version.split('.').map((part: string) => parseInt(part, 10));
+        }
+
+        // Parse the version number strings into arrays
+        const versionArrayA = parseVersion(versionA);
+        const versionArrayB = parseVersion(versionB);
+
+        if (versionArrayA.length !== versionArrayB.length) {
```

Comment:
Can't 3.5.3 > 3.5?

Reply:
Yes, the elements should be the same.
It should be `3.5.3 > 3.5.0`.